AdvLoggerPkg/AdvLoggerPkg.dec

```diff
@@ -81,6 +81,10 @@
   #
   gAdvLoggerPkgTokenSpaceGuid.PcdAdvancedFileLoggerForceEnable|TRUE|BOOLEAN|0x00010184
 
+  ## PcdAdvancedFileLoggerForceEnable - Forces the creation of the Logs subdirectory on non USB devices
+  #
+  gAdvLoggerPkgTokenSpaceGuid.PcdAdvancedLoggerAutoClearEnable|FALSE|BOOLEAN|0x00010188

```

Comment:
Maybe this should be AutoWrap instead of AutoClear?

Reply:
Updated.